universe/Effect.cpp

```diff
@@ -3250,6 +3258,8 @@ void GenerateSitRepMessage::Execute(const ScriptingContext& context) const {
         }
     }
 
+    raw_sitrep_template = raw_sitrep_template && m_affiliation == AFFIL_SELF;
```

Comment:
If it's on the server, then the tech will exist and be active for all players (even if they don't have it in their pedia), and generate sitreps for all of them, won't it? The tech doesn't know which player has the server running on their machine.